packages/@jsii/go-runtime/jsii-experimental/client.go

```diff
@@ -80,38 +113,41 @@ func (c *Client) request(req KernelRequest, res KernelResponse) error {
 	return c.response(res)
 }
 
-func (c *Client) response(res KernelResponse) error {
+func (c *client) response(res KernelResponse) error {
 	if c.reader.More() {
 		return c.reader.Decode(res)
 	}
 
-	errrdr := bufio.NewReader(c.stderr)
-	if errrdr.Size() > 0 {
-		erroutput, err := ioutil.ReadAll(errrdr)
-
-		if err != nil {
-			return err
-		}
-
-		return errors.New(string(erroutput))
-	}
-
```

Comment:
I know @MrArnoldPalmer had wanted a way to queue up errors -- what's the reasoning behind removing this bit?

Reply:
Errors will actually be returned as error messages through `stdout` which you end up processing normally (deserialize from JSON, etc...). `stderr` is actually used for `JSII_DEBUG` logging... In my tests, I have been unable to trigger any debug message to show up with the previous approach, which could *severely* impair the ability to debug problems.